spec/unit/harvestdor/indexer/metrics_spec.rb

```diff
@@ -1,45 +1,45 @@
 require 'spec_helper'
 
 describe Harvestdor::Indexer::Metrics do
-  it "should record successes" do
+  it 'records successes' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.